### PR TITLE
few typos in english version

### PR DESCRIPTION
**Pull request by polofr**
also trailing white spaces removed automatically by editor

### CODE REVIEW

File: effect-types/effect-types.Rmd [hunk 1]
```diff
@@ -136,9 +136,9 @@ ITT<-coef(lm(Y~Z,data=samp))[2]
...
 
-This turns out to be a hard problem (for more on this see this guide). The reasons for non-compliance with treatment can be thought of as an omitted variable. While the receipt of treatment is no-longer independent of potential outcomes, the assignment of treatment status is. As long as random assignment had some positive effect on the probability of receiving treatment, we can use it as an instrument to identify the effects of treatment on the sub-population of subjects who comply with treatment assignment.
+This turns out to be a hard problem (for more on this see [this guide](https://to_be_added)). The reasons for non-compliance with treatment can be thought of as an omitted variable. While the receipt of treatment is no-longer independent of potential outcomes, the assignment of treatment status is. As long as random assignment had some positive effect on the probability of receiving treatment, we can use it as an instrument to identify the effects of treatment on the sub-population of subjects who comply with treatment assignment.
```
Comment (jwbowers) [inline effect-types/effect-types.Rmd]: This should be a link to https://egap.org/resource/10-things-to-know-about-the-local-average-treatment-effect/

Comment (alyssaheinze) [reply effect-types/effect-types.Rmd]: Thanks for your comment.
I just pushed the new changes.

File: power/power.Rmd [hunk 2]
```diff
@@ -185,7 +185,7 @@ Where clustering really causes trouble is when there is a strong relationship be
 
 There are formulas that can help you understand the consequences of clustering — see Gelman/Hill page 447-449 for an extended discussion. While these formulas can be useful, they can also be quite cumbersome to work with. The core insight however is a simple one: you generally get more power from increasing the number of clusters than you do from increasing the number of subjects within clusters. Better to have 100 clusters with 10 subjects in each than 10 clusters with 100 subjects in each.
 
-Again, a more flexible approach to power analysis when dealing with clusters is simulation. See the simulations page [Link here] for starter code, or continue reading Gelman/Hill page 450-453 for another simulation approach.
+Again, a more flexible approach to power analysis when dealing with clusters is simulation. See the (simulations page)[https://TODO] for starter code, or continue reading Gelman/Hill page 450-453 for another simulation approach.
```
Comment (alyssaheinze) [inline power/power.Rmd]: @jwbowers 

Can you tell me which link to put here please? It seems like it never got put in here.
Thank you

Comment (jwbowers) [reply power/power.Rmd]: How about: "See the (Declare Design library for block and cluster randomized experiments)[https://declaredesign.org/r/designlibrary/reference/block_cluster_two_arm_designer.html] for some starter code. The (DeclareDesign)[https://declaredesign.org] software aims to make simulations for power analysis (among many other tasks) easier. See also Gelman/Hill page 450-453 for another simulation approach.

Comment (jwbowers) [reply power/power.Rmd]: @alyssaheinze A response here. Once you'd made that change (if you agree), or something like it. I'll accept the pull request. Thanks so much for discovering these issues!

Comment (alyssaheinze) [reply power/power.Rmd]: @jwbowers 
Just submitted what you suggested.
Sorry for the late reply.

**Review by jwbowers — APPROVED** [review]
Looks great. Thanks so much @alyssaheinze !

**Pull request merged**